bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -35,13 +35,23 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Writer schema version",
+        "uiName": "Writer Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": true,
```

Comment:
@shahsank3t Do we need to set isOptional to false for UI to work correctly? If its optional then there is a chance that the branch user selected first is now deleted in SR. In that case we have seen UI silently failing on that component and getting runtime errors. Can you check the behavior of UI not failing in such scenarios?